### DISCUSSION TITLE
log in

### DISCUSSION THREAD

**Discussion by uwe1805** [Ride-The-Lightning/RTL#1400] [Troubleshooting Questions]
Application locked for 30 minutes due to multiple failed attempts!
Try again after 28/may/2024 15:15:03!

was läuft da verkehrt ? wenn möglich hilfe in deutsch
vielen dank

**Comment by ShahanaFarooqui** [comment]
After 5 incorrect login attempt, the UI locks itself for 30 minutes. If you remember the password (default `password`) correctly then either restart RTL server to reset failed attempts count or wait for 30 minutes for RTL to unlock itself.

If you forgot your password and want to reset it then:
1: Delete `multiPassHashed` key:value from your RTL-Config.json.
2: Add `multiPass` key with your new password in the file.
3: Restart RTL server.
4: Restarting the server will change `multiPass` from plain text to she256 hash and save with `multiPassHashed` key.
5: Use your plain text password value for login through the UI.

Please note that this text has been translated from English to German via google translator.

------------------------------------------------------------------------------------------------------------------------

Nach 5 falschen Anmeldeversuchen sperrt sich die Benutzeroberfläche für 30 Minuten. Wenn Sie sich das Passwort (Standard „Passwort“) richtig merken, starten Sie entweder den RTL-Server neu, um die Anzahl der fehlgeschlagenen Versuche zurückzusetzen, oder warten Sie 30 Minuten, bis RTL sich selbst entsperrt.

Wenn Sie Ihr Passwort vergessen haben und es zurücksetzen möchten, gehen Sie wie folgt vor:
1: Löschen Sie `multiPassHashed` key:value aus Ihrer RTL-Config.json.
2: Fügen Sie den Schlüssel `multiPass` mit Ihrem neuen Passwort in die Datei ein.
3: Starten Sie den RTL-Server neu.
4: Durch einen Neustart des Servers wird `multiPass` von Klartext in she256-Hash geändert und mit dem Schlüssel `multiPassHashed` gespeichert.
5: Verwenden Sie Ihren Klartext-Passwortwert für die Anmeldung über die Benutzeroberfläche.

Bitte beachten Sie, dass dieser Text per Google Übersetzer vom Englischen ins Deutsche übersetzt wurde.
